src/inference/dev_api/openvino/runtime/system_conf.hpp

```diff
@@ -157,6 +200,31 @@ enum ColumnOfProcessorTypeTable {
     PROC_TYPE_TABLE_SIZE = 4   //!< Size of processor type table
 };
 
+/**
+ * @enum       CpuUseStatus
+ * @brief      Defination of CPU_MAP_USED_FLAG column in CPU mapping table.
+ */
+enum CpuUseStatus {
```

Comment:
Maybe it is better to make it enum class. This will allow to avoid using just `ints` for `get_available_cpus` signature

Reply:
Ok, done.